cellprofiler/modules/cropobjects.py

```diff
@@ -51,28 +73,49 @@ def run(self, workspace):
 
         filenames = []
 
-        for label in unique_labels:
-            mask = labels == label
+        if self.export_option == SAVE_MASK:
+            for label in unique_labels:
+                mask = labels == label
+
+                filename = os.path.join(
+                    self.directory.get_absolute_path(),
+                    "{}_{:04d}_{}.tiff".format(self.objects_name.value, label, int(time.time()))
+                )
+
+                skimage.io.imsave(filename, skimage.img_as_ubyte(mask))
+
+                filenames.append(filename)
+
+        if self.export_option == SAVE_PER_OBJECT:
+            orig_image = workspace.image_set.get_image(self.image_name.value)
+            obj_regions = skimage.measure.regionprops(labels)
+            for obj in obj_regions:
+                cropped_image = numpy.copy(orig_image.get_image())
+                top, left, bot, right = obj.bbox
```

Comment:
It would be awesome if this module could support 3D images! `skimage.measure.regionprops` is able to compute the bounding cube.